docs/sql/statements/insert.rst

```diff
@@ -119,14 +137,22 @@ Parameters
 ==========
 
 :table_ident:
-  The identifier (optionally schema-qualified) of an existing table.
+    The identifier (optionally schema-qualified) of an existing table.
 
 :column_ident:
-  The name of a column or field in the table pointed to by *table_ident*.
+    The name of a column or field in the table pointed to by *table_ident*.
 
 :expression:
-  An expression or value to assign to the corresponding column.
+    An expression or value to assign to the corresponding column.
 
 :query:
-  A query (SELECT statement) that supplies the rows to be inserted.
-  Refer to the ``SELECT`` statement for a description of the syntax.
+    A query (SELECT statement) that supplies the rows to be inserted.
+    Refer to the ``SELECT`` statement for a description of the syntax.
+
+:output_expression:
+    An expression to be computed and returned by the INSERT command after each
+    row is updated. The expression can use any column names of the table or
+    ``*`` to return all columns.
```

Comment:
the grammar states this ``* | output_expression``, but the param description states that `output_expression` can be `*` as well, which is not right.

Reply:
?